src/Discovery/DiscoveryConfig.d.ts

```diff
@@ -75,26 +91,36 @@ export interface DiscoveryConfig {
         },
         fieldsToShow: {
             groupName?: string
+            groupWidth?: 'half' | 'full' // defaults to `'half'`
+            // showBackground?: boolean // defaults to `true`
             includeName?: boolean,
             fields: StudyPageFieldConfig[]
         }[]
+        // descriptionField: {
+        //     name: string
+        //     field: string
+        //     includeIfNotAvailable?: boolean // defaults to false
+        //     valueIfNotAvailable?: string[] // defaults to 'n/a'
+        // }
```

Comment:
is this commented block necessary?

Reply:
maybe not, but these (and some other commented fields above) might belongs to some ongping work that @mpingram is planning on, so I'll just keep them in here